cookbooks/clearwater/README.md

```diff
@@ -4,7 +4,8 @@ Description
 Recipes for setting up a Clearwater deployment
 
 The general pattern is to have one recipe per Clearwater component. 
-All components inherit from clearwater-infrastructure
+All components apply local config and inherit from chef-base.
+They do this by inheriting from clearwater-base.
```

Comment:
Is this sentence correct? I think this would be more useful if you explained why each node uses the clearwater-base role